contracts/cheddar_5x5_tic_tac_toe/src/lib.rs

```diff
@@ -124,16 +137,16 @@ impl Contract {
         } else {
             (None, None)
         };
-
         self.available_players.insert(account_id,
             &GameConfig {
                 token_id: AccountId::new_unchecked("near".into()),
```

Comment:
can we remove `token_id` from this structure?

Reply:
Same as above. This will be done in the task where we have two different actions both for deposit and for make_available